packages/mirage/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@bigtest/mirage",
+  "version": "0.0.1",
+  "description": "A client-side server to develop, test and prototype your app.",
+  "main": "dist/index.js",
+  "module": "lib/index.js",
+  "scripts": {
+    "build": "rollup --config",
+    "start": "karma start",
+    "test": "karma start --single-run",
+    "lint": "eslint ./"
+  },
+  "eslintIgnore": [
+    "/dist/"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/thefrontside/bigtest.git"
+  },
+  "keywords": [
+    "test",
+    "mock",
+    "server",
+    "xhr",
+    "xmlhttprequest",
+    "rest",
+    "graphql",
+    "fetch",
+    "react",
+    "ember",
+    "angular"
+  ],
+  "author": "Frontside Engineering <engineering@frontside.io>",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/thefrontside/bigtest/issues"
+  },
+  "homepage": "https://github.com/thefrontside/bigtest/blob/master/packages/mirage/README.md",
+  "devDependencies": {
+    "babel-cli": "^6.24.1",
+    "babel-loader": "^7.1.1",
+    "babel-preset-es2016": "^6.24.1",
+    "eslint": "^4.16.0",
+    "eslint-config-standard": "^11.0.0-beta.0",
+    "eslint-plugin-import": "^2.8.0",
+    "eslint-plugin-node": "^5.2.1",
+    "eslint-plugin-promise": "^3.6.0",
+    "eslint-plugin-standard": "^3.0.1",
+    "karma": "^1.7.0",
+    "karma-chrome-launcher": "^2.2.0",
+    "karma-firefox-launcher": "^1.0.1",
+    "karma-jquery": "^0.2.2",
+    "karma-mocha-reporter": "^2.2.3",
+    "karma-qunit": "^1.2.1",
+    "karma-webpack": "^2.0.4",
+    "qunit": "^1.0.0",
+    "rollup": "^0.53.0",
+    "rollup-plugin-babel": "^4.0.0-beta.0",
+    "rollup-plugin-commonjs": "^8.3.0",
+    "rollup-plugin-node-resolve": "^3.0.0",
+    "webpack": "^3.3.0"
+  },
+  "dependencies": {
+    "faker": "^4.1.0",
+    "inflected": "^2.0.2",
+    "lodash": "^4.17.4",
+    "pretender": "http://github.com/cowboyd/pretender#9ffecb8b789c6162e411ee8f023b5d635129a722"
```

Comment:
What's the reason for this pointing to a sha of a fork?

Reply:
It was because pretender was not compatible with Webpack builds https://github.com/pretenderjs/pretender/pull/209

But this has since been fixed https://github.com/pretenderjs/pretender/pull/216

And I'm not even sure if this is going to be an issue now that we're building with rollup, but in either case, it's been fixed upstream, so we should be able to just use Pretender >  2.0  https://github.com/pretenderjs/pretender/blob/v2.0.0/CHANGELOG.md#20